### PR TITLE
[IO/NIO Task][CCurtis] Counting quantity of words from file

### PR DESCRIPTION
**Pull request by constantin73**

### CODE REVIEW

File: constantinCurtis/src/main/java/inputOutputTask/CountMethod.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        String str;
+        while ((str = bufRead.readLine()) != null) {
+            if (str.equalsIgnoreCase(word)) {
+                result++;
+            }
+        }
+        return result;
+    }
+
+    public static int countNio(String word, Path path) throws IOException {
```
Comment (abadarau) [inline constantinCurtis/src/main/java/inputOutputTask/CountMethod.java:27]: I don't think it's a `countNio`
using `Files.lines` will be more like

File: constantinCurtis/src/main/java/inputOutputTask/Result.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import java.nio.file.Paths;
+
+import static inputOutputTask.CountMethod.countMethod;
+import static inputOutputTask.CountMethod.countNio;
+import static inputOutputTask.WriteTxt.writeNio;
+
+
+public class Result {
+    public static void main(String[] args) throws IOException {
+        File readFromTxt = new File("Edition_1_SOA//constantinCurtis//src//main//resources//wordlist.txt");
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/inputOutputTask/Result.java:15]: please adjust paths

File: constantinCurtis/src/main/java/inputOutputTask/Result.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        File writeResultIo = new File("Edition_1_SOA/constantinCurtis/src/main/resources/Result");
+
+        String findWordIo = "Shelve";
+        String findWordNio = "Abstracted";
+
+        Path wordPath = Paths.get("Edition_1_SOA/constantinCurtis/src/main/resources/wordlist.txt");
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/inputOutputTask/Result.java]: extracts as static final

File: constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class WriteTxt {
+
+    public static void writeToDoc(File file, String text) throws IOException {
+        try {
+            OutputStream out = new BufferedOutputStream(new FileOutputStream(file, true));
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java]: extract from try

File: constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package inputOutputTask;
+
+import java.io.*;
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java:4]: import on demand is evil

File: constantinCurtis/src/main/java/inputOutputTask/Result.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        int quantIo = countMethod(findWordIo, readFromTxt);
+        String textPrintIo = "Word " + findWordIo + " was found " + quantIo + " times.\n";
+
+        WriteTxt writeToText = new WriteTxt();
+        writeToText.writeToDoc(writeResultIo, textPrintIo);
+
+        //Nio reading and writing to result txt
+        int quantNio = countNio(findWordNio, wordPath);
+
+        String textPrintNio = "NIO:Word " + findWordNio + " was found " + quantNio + " times.\n";
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/inputOutputTask/Result.java:34]: please add these statements to count methods

**Review by asecu — COMMENTED** [review]
remove please the 32k lines file from your commit

File: constantinCurtis/src/main/java/inputOutputTask/CountMethod.java (hunk 1 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/CountMethod.java:27]: Thank you, done.

File: constantinCurtis/src/main/java/inputOutputTask/Result.java (hunk 2 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/Result.java:15]: Thanks, fixed

File: constantinCurtis/src/main/java/inputOutputTask/Result.java (hunk 3 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/Result.java]: Thanks, done

File: constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java (hunk 4 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java]: Thank you, changed

File: constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java (hunk 5 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/WriteTxt.java:4]: Agree, changed.

File: constantinCurtis/src/main/java/inputOutputTask/Result.java (hunk 6 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/inputOutputTask/Result.java:34]: Changed, thanks for advice

**Review by abadarau — APPROVED** [review]

**Review by dvmunteanu — APPROVED** [review]

**Review by IIachimov — APPROVED** [review]

**Review by solonarieugeniu — APPROVED** [review]

**Pull request merged**